examples/user_guide/Deploying_Bokeh_Apps.ipynb

```diff
@@ -387,6 +387,7 @@
    "source": [
     "def show_callback():\n",
     "    server.show('/')\n",
+    "TODO: where does loop come from?\n",
```

Comment:
Thanks for catching this, the IOLoop import must have gotten lost somewhere.

Reply:
It used to be:

```python
from tornado.ioloop import IOLoop
loop = IOLoop.current()
```

I wonder if this is needed though or if we can just call ``server.show()`` right after ``server.start()``.